app/models/service_template.rb

```diff
@@ -133,6 +133,9 @@ def create_service(service_task, parent_svc = nil)
     # Hide child services by default
     nh[:display] = false if parent_svc
 
+    # If display is nil, set it to false
+    nh[:display].nil? && nh[:display] = false
```

Comment:
It's more the surrounding context that made me feel like this needed a comment - it's enclosed in other things that require comments so I wanted to make it clear that this was here for a unique reason.

It did occur to me though that I've just hit the same problem with `ServiceTemplate` that I'm trying to solve with `Service` - namely that `nil`s tend to follow you around everywhere. Considered doing the same for `ServiceTemplate`, but felt like I might be doing down a rabbit hole. Any thoughts?